src/textlint-filter-rule-allowlist.js

```diff
@@ -1,17 +1,30 @@
 import path from "path";
-import { rcFile } from "rc-config-loader";
+import yaml from "js-yaml";
 import { getConfigBaseDir } from "@textlint/get-config-base-dir";
 import { matchPatterns } from "@textlint/regexp-string-matcher";
 
+const loadAllowlistConfigFile = (baseDirectory, filePath) => {
+    const fs = require("fs");
```

Comment:
Can you add a comment to source code about it?

It is for suppoting browser bundler.